src/pages/settings/Payments/PaymentMethodList.js

```diff
@@ -0,0 +1,144 @@
+/* eslint-disable react/jsx-props-no-multi-spaces */
```

Comment:
Hmmm I am not seeing why this is needed on this file.

Reply:
I don't think it is. Bad copy/paste :smile: